src/pages/characters/character-create/CharacterCreatePage.js

```diff
@@ -107,7 +107,7 @@ export default function CharacterCreatePage(props) {
 
       <Step.Group size="mini" fluid>
         {stepConfigs.map((stepConfig, configNdx) => (
-          <Step active={step === configNdx} onClick={()  => setStep(configNdx)}>
+          <Step key={stepConfig.title} active={step === configNdx} onClick={()  => setStep(configNdx)}>
```

Comment:
This was throwing a warning in the console. Should have been caught in a previous change.